### DISCUSSION TITLE
tailwind script not having any effect

### DISCUSSION THREAD

**Discussion by SanskarGosavi2003** [tailwindlabs/tailwindcss#7950] [Help]
I am using : 
`"devDependencies": {
    "autoprefixer": "^10.4.4",
    "postcss": "^8.4.12",
    "tailwindcss": "^3.0.23"
  }`
I configured it using:
`module.exports = {
  content: [
    "./src/**/*.{js,jsx}"
  ],
  theme: {
    extend: {},
  },
  plugins: [],
}`
This is my index.css
`@tailwind base;
@tailwind components;
@tailwind utilities;`
And this is my App.js
`function App() {
  return (
    <div class='max-w-sm m-8'>
      <Navbar />
      <Booklist />
    </div>
  );
}`
but the utility classes that I have used up there are not causing any change in my app
P.S. I am learning how to use tailwind with react, and also I am just a beginner in react 
P.S. this is my first doubt on github, please let me know if my syntax of asking is incorrect in any manner

**Comment by 0x15BA88FF** [comment]
hi @SanskarGosavi2003,

I think it has to do with the App.js class: 

```function App() { return ( <div class='max-w-sm m-8'> <Navbar /> <Booklist /> </div> ); }```

try using ```className='max-w-sm m-8'``` instead

**Reply by 0x15BA88FF** [reply]
if it does not get fixed pls create a test repo so we can try and find the issue

**Reply by SanskarGosavi2003** [reply]
Actually i tried both none of them worked

**Reply by SanskarGosavi2003** [reply]
Hi @pascall-de-creator

**Reply by 0x15BA88FF** [reply]
can I have a look at the repo?

**Reply by SanskarGosavi2003** [reply]
Actually i havent done any thing in it yet i just created the react app and tried to implement tailwind so also not uploaded to git hub😥

**Reply by SanskarGosavi2003** [reply]
Also totally unrelated to this but do you know how to upload a react app to git hub with a json database where you fetch data from the json file using commands like" npx json-server --watch data/songs.json --port 7000 "

**Reply by 0x15BA88FF** [reply]
well I think you can make a route to the JSON like as an API so it will run on the same port as the react app
